JSQMessagesViewController/Controllers/JSQMessagesViewController.h

```diff
@@ -162,7 +163,7 @@
  *  you should also override `messagesViewController` to return your
  *  view controller loaded from your custom nib.
  */
-+ (UINib *)nib;
++ (nullable UINib *)nib;
```

Comment:
so.. some of this is non-intuitive.

we should actually remove `nullable` here. it should be nonnull, and we should update the docs to remove mention of `nil`.